use-cases/using-SRIOV-and-kata.md

```diff
@@ -316,3 +317,31 @@ The following example launches a Kata Containers container using SR-IOV:
    ```
    sriov-2:~$ sudo docker run --runtime=kata-runtime --net=vfnet --cap-add SYS_ADMIN --ip=192.168.0.11 -it mcastelino/iperf iperf3 -c 192.168.0.10 bash -c "mount -t ramfs -o size=20M ramfs /tmp; iperf3 -c 192.168.0.10"
    ```
+## Example 2: Launch a Kata Containers container using SR-IOV without SR-IOV Plugin
+
+The second example launches a Kata Containers container using SR-IOV, however doesn't needs an additional plugin. A self build guest kernel, which includes the appropriate NIC driver kernel module, might be necessary. 
```

Comment:
- s/self build/self built/ ?
- Also, could you fold this line into two shorter ones to make future diffs easier to read.
- You can shorten "Kata Containers container" to "Kata Container".